v2/tests/log_sink_console_int/log_sink_console_int.c

```diff
@@ -168,6 +170,20 @@ static void log_sink_console_with_a_context_with_2_properties_works(void)
     // no explicit assert, no crash expected
 }
 
+static void log_sink_console_with_a_context_with_2_properties_works_2(void)
+{
+    // arrange
+    LOG_CONTEXT_LOCAL_DEFINE(test_context, NULL,
+        LOG_CONTEXT_PROPERTY(int32_t, x, 42),
+        LOG_CONTEXT_PROPERTY(bool, y, 1));
```

Comment:
>1 [](http://example.com/codeflow?start=38&length=1)

true.
And also let's have another property with false.

Reply:
changed to true, added another property for false.